lib/http-context.js

```diff
@@ -415,6 +415,7 @@ HttpContext.prototype.setReturnArgByName = function(name, value) {
     // TODO(bajtos) call SharedMethod's convertToBasicRemotingType here?
     this.resultType = typeof returnDesc.type === 'string' ?
       returnDesc.type.toLowerCase() : returnDesc.type;
+    this.isFile = returnDesc.type === 'file';
```

Comment:
Since `this` is the context, the name`isFile` is misleading in my opinion. How about `isReturningFile`?